src/js/game/LevelMVC/LevelBlock.js

```diff
@@ -73,6 +73,12 @@ module.exports = class LevelBlock {
       this.isUsable = true;
     }
 
+    if (blockType === "invisible") {
+      this.isDestroyable = false;
+      this.isUsable = false;
+      this.isWeaklyPowerable = false;
```

Comment:
To have this block clear FoW I think you can set `this.isEmissive = true;`

Reply:
👌